### ISSUE TITLE
Wrong inclusion of tinymce

### ISSUE THREAD

**Issue by Synairgis** [glpi-project/glpi#4955]
**Describe the bug**

As mentioned in the title, I noticed that in the latest version of GLPI (9.3.2), tinymce was wrongly included in the html.class.php. I noticed the problem while I was creating a form using the formcreator plugin.

**To reproduce**

1. Create a form using the formcreator plugin
2. Add a textarea field ("zone de texte" in french)
3. Save everything 
4. Preview the form

**Expected behavior**

Normally, you'd see your field with a tinymce editor appear. Instead, you get a text field and the editor doesn't appear. You also get a bunch of javascript errors in the console.

**Additional context**

Basically, the problem is in the `html.class.php` file line 5744. Here's the line in question:

`$_SESSION['glpi_js_toload'][$name][] = 'lib/tiny_mce/lib/tinymce.js';`

The problem is that the unminified version of the file is no longer there with the latest GLPI version. So the line should instead read:

`$_SESSION['glpi_js_toload'][$name][] = 'lib/tiny_mce/lib/tinymce.min.js';`

**Comment by trasher** [comment]
No, problem is not here. This call ends calling `Html::script()` which takes care of the minified version. If you do not have any issue example in the core, thant it is correct on it side, and problem cames from plugin. 
Closing for now.
